Classes/Command/RunTestsCommand.php

```diff
@@ -54,7 +55,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // Run unit tests
         $runner = new \PHPUnit_TextUI_Command();
-        $result = (int)$runner->run(['test' => $input->getArgument('path')], true);
+        // the first array key is always ignored
+        $optionsForPhpunit = array_merge([0 => ''], explode(' ', $input->getOption('options')));
+
+        // Set default printer only if no specific is set over cli
+        if (!in_array('--printer', $optionsForPhpunit)) {
+            $optionsForPhpunit[] = '--printer';
+            $optionsForPhpunit[] = \OliverKlee\Phpunit\Command\ResultPrinter::class;
```

Comment:
Why full namespace as we are already using namespaces?

Reply:
PhpUnit (the lib from Sebastian Bergmann, not this extension) initialize this class, so full namespace is required.

Reply:
ok

Reply:
ok